binding/identity.go

```diff
@@ -7,45 +7,65 @@ import (
 //
 // Identity API.
 type Identity struct {
-	// hub API client.
-	Client *Client
+	client *Client
 }
 
 //
 // Create a Identity.
 func (h *Identity) Create(r *api.Identity) (err error) {
-	err = h.Client.Post(api.IdentitiesRoot, &r)
+	err = h.client.Post(api.IdentitiesRoot, &r)
 	return
 }
 
 //
 // Get a Identity by ID.
 func (h *Identity) Get(id uint) (r *api.Identity, err error) {
 	r = &api.Identity{}
+	p := Param{
+		Key:   api.Decrypted,
+		Value: "1",
+	}
 	path := Path(api.IdentityRoot).Inject(Params{api.ID: id})
-	err = h.Client.Get(path, r)
+	err = h.client.Get(path, r, p)
+	if err != nil {
+		return
+	}
+	m := r.Model()
+	r.With(m)
 	return
 }
 
 //
 // List Identities.
 func (h *Identity) List() (list []api.Identity, err error) {
 	list = []api.Identity{}
-	err = h.Client.Get(api.IdentitiesRoot, &list)
+	p := Param{
+		Key:   api.Decrypted,
+		Value: "1",
+	}
+	err = h.client.Get(api.IdentitiesRoot, &list, p)
+	if err != nil {
+		return
+	}
+	for i := range list {
```

Comment:
This conversion from resource->model->resource _appears_ to be unnecessary.

Reply:
This is a useless left over from when With() did the decryption.
Fixed.